pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -57,6 +57,58 @@ sb.append('a');     // use this instead
         </example>
     </rule>
 
+    <rule name="AvoidApacheCommonsFileItemNonStreaming"
```

Comment:
Maybe a better name would be `UseIOStreamsWithApacheCommonsFileItem`? IMHO "streaming" immediately hints at the Stream API and not IO streams, so I think it would be clearer